src/Doctrine/Annotation/Token.php

```diff
@@ -0,0 +1,159 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Doctrine\Annotation;
+
+/**
+ * A Doctrine annotation token.
+ */
+class Token
+{
+    const T_NONE = 1;
+    const T_OPEN_DOCBLOCK = 2;
+    const T_CLOSE_DOCBLOCK = 3;
+    const T_ASTERISK = 4;
+    const T_INTEGER = 5;
+    const T_STRING = 6;
+    const T_FLOAT = 7;
+    const T_IDENTIFIER = 8;
+    const T_AT = 9;
+    const T_CLOSE_CURLY_BRACES = 10;
+    const T_CLOSE_PARENTHESIS = 11;
+    const T_COMMA = 12;
+    const T_EQUALS = 13;
+    const T_FALSE = 14;
+    const T_NAMESPACE_SEPARATOR = 15;
```

Comment:
I agree, I did try to use https://github.com/doctrine/annotations in the first place, without success. I'll try harder.